pkg/apis/serving/v1alpha1/revision_validation_test.go

```diff
@@ -69,7 +69,7 @@ func TestConcurrencyModelValidation(t *testing.T) {
 		t.Run(test.name, func(t *testing.T) {
 			got := test.cm.Validate(context.Background())
 			if diff := cmp.Diff(test.want.Error(), got.Error()); diff != "" {
-				t.Errorf("Validate (-want, +got) = %v", diff)
+				t.Errorf("Validate (-want, +got) = %s", diff)
```

Comment:
nit: consistent with below changes?

```suggestion
				t.Errorf("Validate (-want, +got): \n%s", diff)
```